addons/udes_scanned_by_tracking/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "UDES Scanned By Tracking",
+    "version": "11.0",
+    "summary": "Inventory, Logistics, Warehousing",
+    "description": "Extension of udes_stock model for UDES, to add the user to a move line",
+    "depends": ["stock",],
+    "category": "Warehouse",
+    "version": "0.1",
```

Comment:
`version` is listed twice on lines 4 and 9

Reply:
Copy pasta error :facepalm:

Reply:
Fixed: stuck with 11.0 as to track the odoo version is sould be against